api/src/main/java/org/pmiops/workbench/db/model/ConceptSet.java

```diff
@@ -139,7 +138,7 @@ public void setLastModifiedTime(Timestamp lastModifiedTime) {
     this.lastModifiedTime = lastModifiedTime;
   }
 
-  @ElementCollection(fetch = FetchType.LAZY)
+  @ElementCollection(fetch = FetchType.EAGER)
```

Comment:
So this FetchType thing only takes effect when fetching individual entities with children, not lists of them.

If I specify LAZY here, I'll need to add another method that eagerly fetches the concepts under the concept set for the get() method, or fetches it in a second call in a @Transactional block... that didn't seem necessary since we are going to want these concepts when fetching individual concept sets generally.